packages/video_player/video_player/test/video_player_test.dart

```diff
@@ -412,9 +506,9 @@ void main() {
     });
 
     test('calling dispose() on disposed controller does not throw', () async {
-      final VideoPlayerController controller = VideoPlayerController.network(
-        'https://127.0.0.1',
-      );
+      final Uri dummyUri = Uri.parse('https://127.0.0.1');
```

Comment:
The intermediate variable here isn't useful, since it's only used once; it would be better to just inline it:

```
final VideoPlayerController controller =
    VideoPlayerController.networkUrl(Uri.parse('https://127.0.0.1'));
```

Reply:
In fact, since this is used frequently you could make file-level constants:

```
const String _localhost = 'https://127.0.0.1';
final Uri _localhostUri = Uri.parse(_localhost);
```

Then all the tests can use either `_localhost` or `_localhostUri` (depending on the API it's being passed to) instead of repeating the string and/or parsing everywhere.

Reply:
Got it. Just a small question 

Shouldn't be better to keep tests isolated and independents? I mean, even if it's just a constant

Reply:
This thread is still unresolved in the code.

Reply:
Done